README.md

```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
+## Welcome to awesome game of Tetris
 
 A project based learning activity for people who are getting started with Git and GitHub.
 
```

Comment:
Left reviews and line comments
Created a file and made a commit
Opened a pull request